### PR TITLE
Add Hero Component

### PR DESCRIPTION
**Pull request by maribies**
screen shot 2018-06-14 at 11 53 29 am

### CODE REVIEW

File: src/Components/Hero/index.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.Hero {
+  height: 90vh;
+  max-height: 815px;
+  background-size: cover;
+  background: url(http://via.placeholder.com/1200x500) top center;
+}
+
+.FeatureContainer {
```
Comment (joaojusto) [inline src/Components/Hero/index.css]: Where is this rule being used? 🤔

Comment (maribies) [reply src/Components/Hero/index.css]: 👍

**Comment by maribies** [comment]
just going to wait for the nav bar to be completed before continuing.

**Comment by fmmricardo** [comment]
Hi @maribies. can you  please take a look at your component, there are some conflicts that need to be fixed. Check here : https://github.com/thenewdigitalschool/publishing-platform/pull/18/conflicts

**Pull request closed without merging**